index.rst

```diff
@@ -302,107 +305,153 @@ Is this case, we could have something like:
 Nevertheless, the values in the schema are seldom valid beyond a unit testing environment, which requires users to provide some kind of *operational defaults* or *default label*.
-One can see how this can cause confusion when operating the system since now "default" has two different meanings, e.g.; *schema default* and *operational default*.
+One can see how this can cause confusion when operating the system since "default" can be interpreted in two different ways, e.g.; *schema default* and *operational default*.
 Furthermore, it is usually enough to override a small subset of the *schema defaults* for operations.
 Therefore, to get a full set of applied configurations, users must look at two distinct repositories; the configuration repository (for the modified parameters) and the CSC repository (for the schema defaults).
 
 The proposal to improve this aspect of the system is:
 
-#.  Remove default values from the configuration schema.
+#.  Remove all default values from configuration schema definition in the CSC repository.
 
     - See this :download:`example schema <_static/ATSpectrograph_schema.yaml>` for the ATSpectrograph CSC.
+    - Unit tests will need to utilize configuration files stored in the `tests/data/config` directory as is done for the `ATDome CSC <https://github.com/lsst-ts/ts_ATDome/tree/develop/tests/data/config>`_. See `Salobj documentation <https://ts-salobj.lsst.io>`__) for more details.
+
+#.  In the configuration repository for the given CSC (e.g `ts_config_attcs <https://github.com/lsst-ts/ts_config_attcs>`_ for the ATDome) there shall be a ``_init.yaml`` file defining all values that are expected to common to all sites and/or be relatively static in operations (we intentionally use "_init" instead of "_default").
+
+    - See this :download:`example _init.yaml <_static/_init.yaml>` for the ATSpectrograph CSC.
+    - This file is the first configuration file loaded by the CSC
+    - It is not possible to load this file using the `configurationToApply` attribute
+    - Note that all CSCs having multiple algorithms, each with different required attributes, must have an initial set of defaults in this file.
+
+#.  Also in the configuration repository for the given CSC, there shall be a file corresponding to each site where the CSC is used (e.g. ``_summit.yaml, _ncsa.yaml, _base.yaml``).
+    These files contain site specific configuration parameters such as IP addresses and ports.
+    SalObj determines what site file should be loaded automatically by parsing the ``LSST_DDS_PARTITION_PREFIX`` environment variable
+
+    - See this :download:`example _summit.yaml <_static/_summit.yaml>` for the ATSpectrograph CSC.
+    - This file is the second configuration file to get loaded by the CSC and can override any previously declared values.
```

Comment:
`and can override` -> `and overrides` (?)

Reply:
replaced with "and will override"

Reply:
ping?

Reply:
It still says "and can override". I prefer "and will override".